src/stk/utilities/utilities.py

```diff
@@ -449,6 +450,32 @@ def flatten(iterable, excluded_types=None):
             yield x
 
 
+def is_metal_atom(atom):
+    """
+    Check if atom has atomic number of a metal atom.
```

Comment:
Reads a bit strangely to me. Also missing markup around atom.
Maybe something like 
```
Check if `atom` is a metal.
``` 
?